common/src/main/java/com/wynntils/models/mobtotem/MobTotemModel.java

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.models.mobtotem;
+
+import com.wynntils.core.components.Model;
+import com.wynntils.core.components.Models;
+import com.wynntils.handlers.labels.event.EntityLabelChangedEvent;
+import com.wynntils.mc.event.RemoveEntitiesEvent;
+import com.wynntils.models.worlds.WorldStateModel;
+import com.wynntils.utils.mc.McUtils;
+import com.wynntils.utils.mc.type.Location;
+import java.util.HashMap;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import net.minecraft.core.Position;
+import net.minecraft.world.entity.Entity;
+import net.minecraft.world.entity.decoration.ArmorStand;
+import net.minecraft.world.phys.Vec3;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+
+public class MobTotemModel extends Model {
+    private static final Pattern MOB_TOTEM_NAME = Pattern.compile("^§f§l(.*)'s§6§l Mob Totem$");
```

Comment:
I need to check how they render the genitive form for player names ending in `s`. I'm afraid this won't match. I'll check (I'm `mag_icus` so I've fixed a lot of those issues in the past :)) and get back to you.

Reply:
Confirmed that this works fine!